src/components/PluginWallet/PluginWallet.tsx

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { Button } from '../Button';
+import { FiArrowUpRight } from 'react-icons/fi';
+
+export interface PluginWalletProps {
+  isActive: boolean;
+  title: string;
+  iconActive: JSX.Element;
+  iconInactive: JSX.Element;
+  onClickActive: () => void;
+  onClickInactive: () => void;
+}
+
+export function PluginWallet(props: PluginWalletProps) {
+  const {
+    isActive,
+    title,
+    iconActive,
+    iconInactive,
+    onClickActive,
+    onClickInactive,
+  } = props;
+
+  return (
+    <div data-testid="pluginWallet" className="w-full sm:w-80 flex flex-col">
```

Comment:
Yes,  because we need the component to be "responsive" in mobile mode. Otherwise, we have this:
![image](https://github.com/massalabs/ui-kit/assets/91454320/6250d562-5e41-4ae9-80e0-77d9c3d019b1)

Reply:
We are not taking care of that screens size in all others components neither. But it's fine! You can keep as you did. 

Tks for the explanations!